sdks/python/apache_beam/transforms/external.py

```diff
@@ -706,27 +706,12 @@ class JavaJarExpansionService(object):
   This can be passed into an ExternalTransform as the expansion_service
   argument which will spawn a subprocess using this jar to expand the
   transform.
-
-  Args:
```

Comment:
Please don't revert changes to external.py since this feature and doc updates are useful in general.